packages/xo-server-sdn-controller/src/index.js

```diff
@@ -402,6 +521,17 @@ class SDNController extends EventEmitter {
           this._poolNetworks = this._poolNetworks.filter(
             poolNetwork => poolNetwork.network !== networkRef
           )
+
+          forEach(this._crossPoolNetworks, crossPoolNetwork => {
```

Comment:
Avoid using `forEach`, either use the native method if an array or a more specific method like `forOwn`.

Reply:
`this._crossPoolNetworks` is not iterable, that's why I use lodash. I should replace by lodash `forOwn`?

Reply:
IMHO it's better because it documents that we are iterating over the properties of `_crossPoolNetworks`.